locations/spiders/yettel_rs.py

```diff
@@ -0,0 +1,44 @@
+import scrapy
+
+
+class YettelRSSpider(scrapy.Spider):
+    name = "yettel_rs"
+    item_attributes = {
+        "brand": "Yettel",
+        "brand_wikidata": "Q1780171",
+        "country": "RS",
+    }
+
+    def start_requests(self):
+        url = "https://www.yettel.rs/stores/latlong/"
+        yield scrapy.http.FormRequest(url=url, formdata={"initial": "true"}, method="POST")
+
+    def parse(self, response):
+        for index, store in enumerate(response.json()["data"]["stores"]):
+            item["lat"] = store["lat"]
+            item["lon"] = store["lng"]
+
+            item["city"] = store["city"]
+            item["postal_code"] = store["post_number"]
+            item["street_address"] = store["address"]
+
+            oh = opening_hours()
```

Comment:
So this should be the class name:
```suggestion
            oh = OpeningHours()
```

Also I see you've not created `item`.
At the top of the loop, before setting `lat`:
```python
            item = Feature()
```

We then need to import both of these classes, so at the top of the file:
```python
from locations.hours import OpeningHours
from locations.items import Feature
```